serverless.yml

```diff
@@ -15,12 +15,48 @@ provider:
     - Effect: "Allow"
       Action:
         - "cloudformation:DescribeStack*"
+        - "dynamodb:DescribeTable"
+        - "dynamodb:Query"
+        - "dynamodb:Scan"
+        - "dynamodb:GetItem"
+        - "dynamodb:PutItem"
+        - "dynamodb:UpdateItem"
+        - "dynamodb:DeleteItem"
       Resource: "*"
```

Comment:
Can we make this the specific table name?
Usually helps to set something like the table name in a custom serverless variable, so you can reference it in places like this.

i.e.
`Resource: arn:aws:dynamodb:*:*:table/${self:custom.tableName}`

in serverless.yml you can just put in something like:
`
custom:
  tableName: stackJanitorTable
`

Reply:
bumping this issue.

Reply:
ok